heft-plugins/heft-workbox-plugin/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./node_modules/@rushstack/heft-node-rig/profiles/default/tsconfig-base.json",
+
+  "compilerOptions": {
+    "types": ["node"],
+    "lib": ["WebWorker"]
```

Comment:
What uses WebWorker?

Reply:
ServiceWorker code requires the WebWorker lib due to the way the TS declarations are done.